front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/MostReactedIdeasWidget/Ideas/index.tsx

```diff
@@ -2,61 +2,47 @@ import React from 'react';
 
 // components
 import { Box } from '@citizenlab/cl2-component-library';
-import IdeaCard from './IdeaCard';
+import IdeaCard from '../../SingleIdeaWidget/IdeaCard';
```

Comment:
I suppose it makes more sense to import an idea card in "Most reacted ideas" from "Single idea" than in "Single idea" from "Most reacted ideas".